sensor/models.py

```diff
@@ -209,18 +209,22 @@ def save(self, *args, **kwargs):
     def error(cls, data):
         if data is None:
             return "Error: empty payload"
-
+    
         missing_keys = []
-        for key in ["key", "sensorid", "value", "timestamp"]:
+        for key in ["key", "value", "timestamp"]:
             if not key in data:
                 missing_keys.append(key)
+    
+        if not "sensor_id" or not "id" in data:
```

Comment:
please use `'id' not in data` over `not 'id' in data` as per Python conventions.